packages/twenty-front/src/modules/settings/data-model/graph-overview/components/SettingsDataModelOverviewObject.tsx

```diff
@@ -100,36 +99,38 @@ const StyledObjectLink = styled(Link)`
 `;
 
 export const SettingsDataModelOverviewObject = ({
-  data,
+  data: objectMetadataItem,
```

Comment:
Could we rename this props to objectMetadataItem ?

Reply:
This component props are built using the `NodeProps` generic that encapsulate any custom payload within the `data` property.
```ts
type SettingsDataModelOverviewObjectProps =
  NodeProps<SettingsDataModelOverviewObjectNode>;
```

In the other hand it seems like it does not consume any other props than the `data` one

Reply:
Ok this zone could be improved but it's not critical.